<launch>
  <node pkg="localizer" type="localizer_node" name="localizer" output="screen">
    <param name="img_width"  value="1292"/>
    <param name="img_height" value="964"/>

    <param name="pose_topic" value="/slam_out_pose"/>
    <param name="image_topic" value="/camera/image_raw"/>
    <param name="map_file" value="/home/andre-criis/map.txt"/>

    <param name="detector_th" value="0.35"/>
    <param name="scaler" value="10"/>
    <param name="model_path"
      value="/home/andre-criis/Source/coral/detect/train/train_res/mv1_11-11/output_tflite_graph_edgetpu.tflite"/>
    <param name="labels_path"
      value="/home/andre-criis/Source/coral/detect/train/train_res/mv1_11-11/labels.txt"/>
  </node>
</launch>
